.golangci.yml

```diff
@@ -98,6 +99,5 @@ issues:
       text: 'G101: Potential hardcoded credentials'
     - path: pkg/controller/elasticsearch/driver/version\.go
       text: 'Consider preallocating `vs`'
-    - linters:
-        - stylecheck
-      text: 'ST1016: methods on the same type should have the same receiver name.*"in".*'
+    - path: cmd/manager/main\.go
```

Comment:
Again, this list was roughly sorted by path. No big deal though.

Reply:
Ah I didn't realize for either of them